bin/yaml/dart.yaml

```diff
@@ -19,3 +20,8 @@ compilers:
       - 2.10.5
       - 2.9.3
       - 2.8.4
+    nightly:
+      if: nightly
+      url: https://storage.googleapis.com/dart-archive/channels/{name}/release/latest/sdk/dartsdk-linux-x64-release.zip
```

Comment:
So, as it stands this will download only once (we by default don't download stuff again). There are special "nightly" `type` for some things, but not for `ziparchive`; so a workaround would be to add `install_always: true` which unconditionally installs over the top every time.

Reply:
Thanks, I set `install_always` to `true`.